src/content/docs/data-apis/manage-data/view-system-limits.mdx

```diff
@@ -31,14 +31,14 @@ What the color-coding in the violation table means:
 
 Some tips on using the limits UI: 
 * To get more detail about a table entry, try clicking it. Some entries have more detail, including associated NRQL queries.   
-* When you select a time range greater than 6 hrs, the chart uses average values, which may be smoothed out and not show spikes. For longer ranges, you may see the message "For the time window chosen, the 'Max limit usage' value represents the average of your usage limits." To see more accurate results, use the time picker to zoom into a time range that's less than 6 hours.
+* When you select a time range greater than 6 hrs, the chart uses average values, which may smooth out spikes. This is the reason you may see the message "For the time window chosen, the 'Max limit usage' value represents the average of your usage limits." To see more accurate results, use a below-6-hours time range.
```

Comment:
I know that when numbers are <10, you spell them out (6 --> six), but there might be an exception when talking about time intervals for data. Just calling it out and not changing it in case there's a UI-related reason for keeping the numeral.